components/Form.js

```diff
@@ -0,0 +1,85 @@
+import styled from "styled-components";
+import { StyledButton } from "@/components/StyledButton";
+import { useRouter } from "next/router";
+
+export default function Form() {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    handleClick(event);
+  }
+
+  const router = useRouter();
+
+  const handleClick = (event) => {
+    event.preventDefault();
+    router.push("https://i.imgur.com/7KpCS0Y.jpg");
+  };
+
+  return (
+    <FormContainer onSubmit={handleSubmit}>
+      <Input
+        id="firstKeyword"
+        name="firstKeyword"
+        type="text"
+        placeholder="FIRST"
+        oninvalid="this.setCustomValidity('TYPE HERE IDIOT')"
+        oninput="setCustomValidity('')"
```

Comment:
does this really work as intended?

when I submit the form without any input, I get the normal message from the browser and not your custom message.

Reply:
no this was a leftover, thanks